Microsoft.Toolkit.Uwp.UI.Controls.Design/MetadataRegistration.cs

```diff
@@ -4,43 +4,41 @@
 
 using System;
 using System.Reflection;
-using Microsoft.Windows.Design.Metadata;
 using Microsoft.Toolkit.Uwp.Design.Common;
+using Microsoft.Toolkit.Uwp.UI.Controls.Design;
+
+#if VS_DESIGNER_PROCESS_ISOLATION
+using Microsoft.Windows.Design.Metadata;
+#else
+using Microsoft.VisualStudio.DesignTools.Extensibility.Metadata;
+#endif
```

Comment:
This is only for compat reasons with Visual Studio targets pre 16299 where they used a different approach to the designer project right?

Reply:
Yes

Reply:
The Toolkit only supports 16299 and above currently, and we're going to bump that up higher (see #3440). So do we need to worry about this?

Reply:
This change is for VS XAML Designer and not Windows revision specific. As long as devs don't need to hang on to the older VS versions we can safely remove this in the future.